practice-three/src/js/script.js

```diff
@@ -4,6 +4,8 @@ const emailInput = document.getElementById('email');
 const salaryInput = document.getElementById('salary');
 const cityInput = document.getElementById('city');
 const submitBtn = document.getElementById('submitBtn');
+const userTableBody = document.getElementById('userTableBody')
+let courseApi = 'http://localhost:3000/userDatabase'
```

Comment:
Why did you use `let`? And this api is related to User, `courseApi` doesn't make sense.